flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -4056,4 +4056,15 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "f55=f57",
       "true")
   }
+
+  @Test
+  def testCast(): Unit = {
```

Comment:
Could you add this test to `ScalarOperatorsTest`? Because `cast` is an operator. 
Besides, coul dyou add cast tests for binary and varbinary source fields?

Reply:
Yes, Thank you for reminder. I updated the test case.